tests/parser/tst_parser.cpp

```diff
@@ -112,10 +112,10 @@ static CborError qstring_printf(void *out, const char *fmt, ...)
     int buf_size = vsnprintf(&c, 1, fmt, va) + 1;
     va_end(va);
 
-    if (buf_size < 0)
+    if (buf_size <= 0)
```

Comment:
This is wrong. vsnprintf returns negative on error; zero is not an error.

Reply:
Well, in this case you're already doing a +1 on line 112.  A buf_size of 0 means vsnprintf returned -1.

Reply:
Oh, right. Good point.